src/responder/component.ts

```diff
@@ -230,7 +230,7 @@ export class MultiResponderComponent extends Component<ResponderAnchorState, Blo
                     await this.responder.txMined(action.identifier, action.nonce);
                     break;
                 case ResponderActionKind.EndResponse:
-                    this.responder.endResponse(action.appointmentId);
+                    await this.responder.endResponse(action.appointmentId);
                     break;
                 default:
                     throw new ArgumentError("Unrecognised action kind.", action);
```

Comment:
I previously added an `UnreachableCaseError` for this kind of things, if you like to use it; has the advantage of being type safe at compile-time.

Reply:
Now using UnreacheableCaseError